moto/secretsmanager/models.py

```diff
@@ -39,11 +46,23 @@ def get_secret_value(self, secret_id, version_id, version_stage):
 
         return response
 
-    def secret_arn(self):
+    def create_secret(self, name, secret_string, **kwargs):
+
+        response = json.dumps({
+            "ARN": self.secret_arn(self.region, name),
+            "Name": name,
+            "VersionId": "A435958A-D821-4193-B719-B7769357AER4",
+        })
+
+        return response
+
+    def secret_arn(self, region, secret_id):
         return "arn:aws:secretsmanager:{0}:1234567890:secret:{1}-rIjad".format(
-            self.region, self.secret_id)
+            region, secret_id)
 
 
-available_regions = boto3.session.Session().get_available_regions("secretsmanager")
-print(available_regions)
-secretsmanager_backends = {region: SecretsManagerBackend(region_name=region) for region in available_regions}
+available_regions = (
```

Comment:
Are the brackets necessary?

Reply:
Just the parentheses works. I use them because of the PEP8 79 characters line length limit.

Reply:
But if you don't mind the long line, I can remove the parentheses.